cardano_node_tests/tests/tests_conway/test_treasury_withdrawals.py

```diff
@@ -355,6 +356,21 @@ def _cast_vote(
             governance_utils.check_vote_view(cluster_obj=cluster, vote_data=voted_votes.cc[0])
         governance_utils.check_vote_view(cluster_obj=cluster, vote_data=voted_votes.drep[0])
 
+        if configuration.HAS_DBSYNC:
```

Comment:
How about extracting this code to its own function and moving it to `dbsync_utils.py` ?